<ui version="4.0" >
 <class>ImaplibManagementWidget</class>
 <widget class="QWidget" name="ImaplibManagementWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>141</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="KPushButton" name="defaultButton" >
     <property name="text" >
      <string>Set Default</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="KPushButton" name="removeButton" >
     <property name="text" >
      <string>R&amp;emove</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="KPushButton" name="editButton" >
     <property name="text" >
      <string>&amp;Modify...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KPushButton" name="addButton" >
     <property name="text" >
      <string>A&amp;dd...</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="0" column="0" >
    <widget class="QTreeWidget" name="transportList" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
